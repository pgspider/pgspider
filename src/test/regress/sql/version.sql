--
-- VERSION
--

-- ===================================================================
-- test --version option
-- ===================================================================

\! clusterdb \-\-version
\! createdb \-\-version
\! createuser \-\-version
\! dropdb \-\-version
\! dropuser \-\-version
\! ecpg \-\-version
\! initdb \-\-version
\! pg_archivecleanup \-\-version
\! pg_basebackup \-\-version
\! pgbench \-\-version
\! pg_checksums \-\-version
\! pg_config \-\-version
\! pg_controldata \-\-version
\! pg_ctl \-\-version
\! pg_dump \-\-version
\! pg_dumpall \-\-version
\! pg_isready \-\-version
\! pg_receivewal \-\-version
\! pg_recvlogical \-\-version
\! pg_resetwal \-\-version
\! pg_restore \-\-version
\! pg_rewind \-\-version
\! pgspider \-\-version
\! pg_test_fsync \-\-version
\! pg_test_timing \-\-version
\! pg_upgrade \-\-version
\! pg_verifybackup \-\-version
\! pg_waldump \-\-version
\! postmaster \-\-version
\! psql \-\-version
\! reindexdb \-\-version
\! vacuumdb \-\-version
\! $PWD/../../../contrib/oid2name/oid2name \-\-version
\! $PWD/../../../contrib/pg_standby/pg_standby \-\-version
\! $PWD/../../../contrib/vacuumlo/vacuumlo \-\-version
\! ./pg_regress \-\-version
-- src/test/isolation/isolationtester not support --version option
-- \! $PWD/../isolation/isolationtester \-\-version
\! $PWD/../isolation/pg_isolation_regress \-\-version

-- ===================================================================
-- test --V option
-- ===================================================================

\! clusterdb \-V
\! createdb \-V
\! createuser \-V
\! dropdb \-V
\! dropuser \-V
\! ecpg \-V
\! initdb \-V
\! pg_archivecleanup \-V
\! pg_basebackup \-V
\! pgbench \-V
\! pg_checksums \-V
-- pg_config have no -V option
-- \! pg_config \-V
\! pg_controldata \-V
\! pg_ctl \-V
\! pg_dump \-V
\! pg_dumpall \-V
\! pg_isready \-V
\! pg_receivewal \-V
\! pg_recvlogical \-V
\! pg_resetwal \-V
\! pg_restore \-V
\! pg_rewind \-V
\! pgspider \-V
\! pg_test_fsync \-V
\! pg_test_timing \-V
\! pg_upgrade \-V
\! pg_verifybackup \-V
\! pg_waldump \-V
\! postmaster \-V
\! psql \-V
\! reindexdb \-V
\! vacuumdb \-V
\! $PWD/../../../contrib/oid2name/oid2name \-V
\! $PWD/../../../contrib/pg_standby/pg_standby \-V
\! $PWD/../../../contrib/vacuumlo/vacuumlo \-V
\! ./pg_regress \-V
\! $PWD/../isolation/isolationtester \-V
\! $PWD/../isolation/pg_isolation_regress \-V

-- ===================================================================
-- test function version()
-- ===================================================================
SELECT substring(version(), 1, 13) AS version;
SELECT * FROM substring(version(),1,13);
