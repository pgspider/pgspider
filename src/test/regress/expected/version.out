--
-- VERSION
--
-- ===================================================================
-- test --version option
-- ===================================================================
\! clusterdb \-\-version
clusterdb (PGSpider) 13.0
\! createdb \-\-version
createdb (PGSpider) 13.0
\! createuser \-\-version
createuser (PGSpider) 13.0
\! dropdb \-\-version
dropdb (PGSpider) 13.0
\! dropuser \-\-version
dropuser (PGSpider) 13.0
\! ecpg \-\-version
ecpg (PGSpider) 13.0
\! initdb \-\-version
initdb (PGSpider) 13.0
\! pg_archivecleanup \-\-version
pg_archivecleanup (PGSpider) 13.0
\! pg_basebackup \-\-version
pg_basebackup (PGSpider) 13.0
\! pgbench \-\-version
pgbench (PGSpider) 13.0
\! pg_checksums \-\-version
pg_checksums (PGSpider) 13.0
\! pg_config \-\-version
PGSpider 13.0
\! pg_controldata \-\-version
pg_controldata (PGSpider) 13.0
\! pg_ctl \-\-version
pg_ctl (PGSpider) 13.0
\! pg_dump \-\-version
pg_dump (PGSpider) 13.0
\! pg_dumpall \-\-version
pg_dumpall (PGSpider) 13.0
\! pg_isready \-\-version
pg_isready (PGSpider) 13.0
\! pg_receivewal \-\-version
pg_receivewal (PGSpider) 13.0
\! pg_recvlogical \-\-version
pg_recvlogical (PGSpider) 13.0
\! pg_resetwal \-\-version
pg_resetwal (PGSpider) 13.0
\! pg_restore \-\-version
pg_restore (PGSpider) 13.0
\! pg_rewind \-\-version
pg_rewind (PGSpider) 13.0
\! pgspider \-\-version
pgspider (PGSpider) 13.0
\! pg_test_fsync \-\-version
pg_test_fsync (PGSpider) 13.0
\! pg_test_timing \-\-version
pg_test_timing (PGSpider) 13.0
\! pg_upgrade \-\-version
pg_upgrade (PGSpider) 13.0
\! pg_verifybackup \-\-version
pg_verifybackup (PGSpider) 13.0
\! pg_waldump \-\-version
pg_waldump (PGSpider) 13.0
\! postmaster \-\-version
pgspider (PGSpider) 13.0
\! psql \-\-version
psql (PGSpider) 13.0
\! reindexdb \-\-version
reindexdb (PGSpider) 13.0
\! vacuumdb \-\-version
vacuumdb (PGSpider) 13.0
\! $PWD/../../../contrib/oid2name/oid2name \-\-version
oid2name (PGSpider) 13.0
\! $PWD/../../../contrib/pg_standby/pg_standby \-\-version
pg_standby (PGSpider) 13.0
\! $PWD/../../../contrib/vacuumlo/vacuumlo \-\-version
vacuumlo (PGSpider) 13.0
\! ./pg_regress \-\-version
pg_regress (PGSpider) 13.0
-- src/test/isolation/isolationtester not support --version option
-- \! $PWD/../isolation/isolationtester \-\-version
\! $PWD/../isolation/pg_isolation_regress \-\-version
pg_regress (PGSpider) 13.0
-- ===================================================================
-- test --V option
-- ===================================================================
\! clusterdb \-V
clusterdb (PGSpider) 13.0
\! createdb \-V
createdb (PGSpider) 13.0
\! createuser \-V
createuser (PGSpider) 13.0
\! dropdb \-V
dropdb (PGSpider) 13.0
\! dropuser \-V
dropuser (PGSpider) 13.0
\! ecpg \-V
ecpg (PGSpider) 13.0
\! initdb \-V
initdb (PGSpider) 13.0
\! pg_archivecleanup \-V
pg_archivecleanup (PGSpider) 13.0
\! pg_basebackup \-V
pg_basebackup (PGSpider) 13.0
\! pgbench \-V
pgbench (PGSpider) 13.0
\! pg_checksums \-V
pg_checksums (PGSpider) 13.0
-- pg_config have no -V option
-- \! pg_config \-V
\! pg_controldata \-V
pg_controldata (PGSpider) 13.0
\! pg_ctl \-V
pg_ctl (PGSpider) 13.0
\! pg_dump \-V
pg_dump (PGSpider) 13.0
\! pg_dumpall \-V
pg_dumpall (PGSpider) 13.0
\! pg_isready \-V
pg_isready (PGSpider) 13.0
\! pg_receivewal \-V
pg_receivewal (PGSpider) 13.0
\! pg_recvlogical \-V
pg_recvlogical (PGSpider) 13.0
\! pg_resetwal \-V
pg_resetwal (PGSpider) 13.0
\! pg_restore \-V
pg_restore (PGSpider) 13.0
\! pg_rewind \-V
pg_rewind (PGSpider) 13.0
\! pgspider \-V
pgspider (PGSpider) 13.0
\! pg_test_fsync \-V
pg_test_fsync (PGSpider) 13.0
\! pg_test_timing \-V
pg_test_timing (PGSpider) 13.0
\! pg_upgrade \-V
pg_upgrade (PGSpider) 13.0
\! pg_verifybackup \-V
pg_verifybackup (PGSpider) 13.0
\! pg_waldump \-V
pg_waldump (PGSpider) 13.0
\! postmaster \-V
pgspider (PGSpider) 13.0
\! psql \-V
psql (PGSpider) 13.0
\! reindexdb \-V
reindexdb (PGSpider) 13.0
\! vacuumdb \-V
vacuumdb (PGSpider) 13.0
\! $PWD/../../../contrib/oid2name/oid2name \-V
oid2name (PGSpider) 13.0
\! $PWD/../../../contrib/pg_standby/pg_standby \-V
pg_standby (PGSpider) 13.0
\! $PWD/../../../contrib/vacuumlo/vacuumlo \-V
vacuumlo (PGSpider) 13.0
\! ./pg_regress \-V
pg_regress (PGSpider) 13.0
\! $PWD/../isolation/isolationtester \-V
isolationtester (PGSpider) 13.0
\! $PWD/../isolation/pg_isolation_regress \-V
pg_regress (PGSpider) 13.0
-- ===================================================================
-- test function version()
-- ===================================================================
SELECT substring(version(), 1, 13) AS version;
    version    
---------------
 PGSpider 13.0
(1 row)

SELECT * FROM substring(version(),1,13);
   substring   
---------------
 PGSpider 13.0
(1 row)

